src/main/java/ch/bfh/bti7081/s2018/green/views/MedciationPrescriptionView.java

```diff
@@ -0,0 +1,179 @@
+package ch.bfh.bti7081.s2018.green.views;
+
+import ch.bfh.bti7081.s2018.green.layouts.UiElementHasValue;
+import ch.bfh.bti7081.s2018.green.models.entities.Medication;
+import ch.bfh.bti7081.s2018.green.models.entities.Patient;
+import ch.bfh.bti7081.s2018.green.models.entities.Staff;
+import ch.bfh.bti7081.s2018.green.views.dummyValueGenerators.MedDummyList;
+import ch.bfh.bti7081.s2018.green.views.dummyValueGenerators.PatDummy;
+import ch.bfh.bti7081.s2018.green.views.interfaces.IRetSetAbstField;
+import com.vaadin.navigator.View;
+import com.vaadin.ui.*;
+import java.time.LocalDateTime;
+import java.util.*;
+
+public class MedciationPrescriptionView extends CustomLayout implements View, IRetSetAbstField {
+
+    public Medication getMedPrescibed() {
+        return medPrescibed;
+    }
+
+    private List<UiElementHasValue> listUiElements = new ArrayList<>();
+
+    protected Window window;
+
+    public UiElementHasValue getMedPeriod() {
+        return medPeriod;
+    }
+
+    public UiElementHasValue getMedDose() {
+        return medDose;
+    }
+
+    public UiElementHasValue getMedPrescriberFullName() {
+        return medPrescriberFullName;
+    }
+
+    public UiElementHasValue getMedRecordCreated() {
+        return medRecordCreated;
+    }
+
+    public UiElementHasValue getMedRecordModified() {
+        return medRecordModified;
+    }
+
+    public UiElementHasValue getMedStartDate() {
+        return medStartDate;
+    }
+
+    public UiElementHasValue getMedEndDate() {
+        return medEndDate;
+    }
+
+    public UiElementHasValue getMedName() {
+        return medName;
+    }
+
+    UiElementHasValue medName;
+    UiElementHasValue medStartDate;
+    UiElementHasValue medEndDate;
+    UiElementHasValue medPeriod;
+    UiElementHasValue medDose;
+    UiElementHasValue medPrescriberFullName;
+    UiElementHasValue medRecordCreated;
+    UiElementHasValue medRecordModified;
+
+    Medication medPrescibed;
+    //List<UiElementHasValue> txtToAdd;
+
+    public Patient getPatDummy() {
+        return PatDummy;
+    }
+
+    public Staff getStaffDummy() {
+        return StaffDummy;
+    }
+
+    Patient PatDummy = MedDummyList.buildDummyMedList(ch.bfh.bti7081.s2018.green.views.dummyValueGenerators.PatDummy.getPatDummy()).get(0).getPatient();
```

Comment:
Remove dummy data. If you need data, then add it to data.sql

Reply:
I use data from the DB... i need a way how to know what patient is select and what doctor is logged in to have a functional Medication app.

Something similar as one could do in PHP for "$_Session"?
But I see if I uncomment code as in Comment above, it's probably ok because accessible after "initializeClasses" in NavigatorUI,